FelipeJimenez.md

```diff
@@ -0,0 +1 @@
+_Name_: Felipe Alejandro Jimenez Castillo
```

Comment:
Please change `name` from Italic to Blod.

Reply:
Resolved in https://github.com/denso666/team-work/commit/3461203e995a381ebc16a05d6cd9fa7648c6f6ab .